src/senaite/sync/utils.py

```diff
@@ -28,6 +30,22 @@ def to_review_history_format(review_history):
     return review_history
 
 
+def is_item_allowed(item):
+    """
+    Check if an item can be handled based in its portal type.
+    :return:
+    """
+    if not isinstance(item, dict):
+        return False
+
+    portal_types = api.get_tool("portal_types")
+    pt = item.get("portal_type", "SKIP")
+    if pt in SKIP_PORTAL_TYPES or pt not in portal_types:
```

Comment:
I think you do not have to prefill `SKIP_PORTAL_TYPES ` with "SKIP" because you get `None` in line 42, which shouldn't be in `portal_types`;)